### ISSUE TITLE
Benchmarks?

### ISSUE THREAD

**Issue by dgryski** [Quentin-M/pqueue#1]
I think benchmarks against a slice-based implementation using `container/heap` would be interesting.  My implementation of Hollow Heaps ( https://github.com/dgryski/go-hollow ) for example is much slower due to all the pointer chasing.

**Comment by Quentin-M** [comment]
Absolutely! I had the desire to benchmark it too but lacked references. Thanks.
